src/lib/components/Subscribe/index.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @fileoverview Element that renders newsletter subscription form.
+ */
+
+import {BaseElement} from "../BaseElement";
+import "./_styles.scss";
+
+/**
+ * Element that renders newsletter subscription form.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class Subscribe extends BaseElement {
+  constructor() {
+    super();
+
+    this.form = this.querySelector("form");
+    this.subscribeError = this.querySelector("div#w-subscribe__error");
```

Comment:
Don't think you need `div` in this selector. I think using either the id or a class is fine here. I tend to never use id's unless I'm using shadow dom but that's kinda just me being dogmatic :P